pyproject.toml

```diff
@@ -12,6 +12,7 @@ requires-python = ">=3.11"
 dependencies = [
   "click",
   "redis",
+  "requests",
```

Comment:
I like using https://github.com/encode/httpx for my new projects - as it is faster and supports async.